save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -170,10 +170,17 @@ class OrganizationService(
      * @throws NoSuchElementException
      */
     fun saveAvatar(name: String, relativePath: String) {
```

Comment:
Shouldn't it be marked as `Transactional`?

Reply:
No. This annotation is needed when we have more than one method that changes the data in the database. In all other cases, it is bad to keep an open transaction to the database.

Reply:
oh ok